packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -225,7 +225,7 @@ export async function batchInstallFontFaces( fontFamilyId, fontFacesData ) {
 			// Handle network errors or other fetch-related errors
 			results.errors.push( {
 				data: fontFacesData[ index ],
-				message: `Fetch error: ${ result.reason.message }`,
+				message: result.reason.message,
```

Comment:
👍 to passing the real error message without the prefix.

Is it worth guarding against empty reason messages and providing a fallback such as `Unknown error`?